app/javascript/src/translation/en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "note": {
+    "delete": {
+      "successWithCount_one": "{{count}} note is deleted successfully!",
+      "successWithCount_other": "{{count}} notes was deleted successfully!"
```

Comment:
- First of all, the naming convention is not right. It ought to be camelCase.
-  Passing count as a prop to the translation is an anti-pattern. Because what would be shown when a single note is deleted? `1 note is deleted` is redundant. Rather a simple `Note has been deleted` would do when it's a single note.
-  Use https://www.i18next.com/translation-function/plurals.

Reply:
@yedhink And when it's plural, will "Notes have been deleted successfully" be enough without the count?

Reply:
> Passing count as a prop to the translation is not the pattern that we follow in BB.
On second thought, the above statement answers the question.

Reply:
> Passing count as a prop to the translation is not the pattern that we follow in BB.
On second thought, the above statement answers the question.